src/resources/general/compiler-development.md

```diff
@@ -0,0 +1,43 @@
+# 🛠️ Compiler Development
```

Comment:
Please move this file and the other one back to `advanced/`, that's their intended home. If you want to fix the 404 you were seeing that can be done in `config.mts`, ideally as a separate PR :)

Reply:
Gotcha. Is the "Floating Point Numbers page" intended to be in the General Resource category or the Advanced Resource category? I feel like floating point numbers are pretty basic and should not be in the advanced category (especially based on the current content of this page).

Reply:
It is intended to be in advanced for now. Currently everything on that page is more advanced than what we'll have in the general category, but that may change in the future

Reply:
Actually, looking back, fine to leave it in general. `float.exposed` is a super helpful tool and not too advanced.

Reply:
> Actually, looking back, fine to leave it in general. `float.exposed` is a super helpful tool and not too advanced.

That's what I was saying. It is currently linked to general in the URL, but router to advance resource category.
![image](https://github.com/user-attachments/assets/344fc319-a625-4c8d-b578-dc5e13c18ed4)